scripts/base/init-bare.zeek

```diff
@@ -2895,6 +2897,8 @@ export {
 		session_key : string &optional;
 		## The Windows version information, if supplied
 		version     : NTLM::Version &optional;
+
+		ntlmv2response    : string &optional;
```

Comment:
Could this be named just `response` instead?

Reply:
Done :)